test/cat.jl

```diff
@@ -249,8 +302,6 @@ end
     @test err.value.msg == "column(s) B are missing from argument(s) 1"
     # multiple missing 1 column
     err = @test_throws ArgumentError vcat(df1, df2, df2, df2, df2, df2)
-    err2 = @test_throws ArgumentError reduce(vcat, [df1, df2, df2, df2, df2, df2])
```

Comment:
@oxinabox I think my rebase may have collided with your optimization for `reduce(vcat, dfs)`. Should this throw an error or not?

Reply:
If `vcat(xs...)` throws an error, then `reduce(vcat, xs)` should throw the same error.
If not, then not.

Reply:
Okay then this error should remain deleted. It's good to go.

Reply:
I missed what this error was about. I have added it back in.